torch/testing/_internal/opinfo/core.py

```diff
@@ -2103,6 +2156,7 @@ def __init__(
         supports_rhs_python_scalar=True,  # Whether the operator allows Tensor x scalar inputs
         supports_one_python_scalar=False,  # Whether the operator allows scalar x tensor and tensor x scalar inputs
         supports_two_python_scalars=False,  # Whether the operator allows scalar x scalar inputs
+        skipXPU=True,
```

Comment:
Can we replace it with "skipDevices=[xpu]' ?

Reply:
Updated the skip mechanism:
In OpInfo class define two fields:
```
   # skip the test for a device
    skip_device: Tuple = tuple()

    # enable the test for a device
    enable_skipped_device: Tuple = tuple()
```
By default skip_device=('xpu', ), if enable_skipped_device=('xpu') is defined in op_db, xpu will be enabled, otherwise xpu will be added to skips.
```
self.skip_device = ('xpu',)
        for device in (set(self.skip_device).difference(set(self.enable_skipped_device))):
            if self.skips is not None:
                self.skips = (*self.skips, DecorateInfo(unittest.skip, device_type=device, dtypes=None))
            else:
                self.skips = (DecorateInfo(unittest.skip, device_type=device, dtypes=None))
````

Reply:
Design is changed by using a util function and xpu_op_db.yaml to skip unsupported ops.